src/lib/y2partitioner/widgets/encrypt.rb

```diff
@@ -224,6 +224,26 @@ def help_for_luks2
         )
       end
 
+      # Help text for the Regular systemd_fde encryption method
+      #
+      # @return [String]
+      def help_for_systemd_fde
+        encrypt_method = Y2Storage::EncryptionMethod.find(:systemd_fde)
+
+        format(
+          # TRANSLATORS: help text for Regular Luks2 encryption method together wit TPM2 support
+          _("<p><b>%{label}</b>: allows to encrypt the device using LUKS2. "\
+            "You have to provide the encryption password and the password-based key derivation " \
+            "function (PBKDF) that will be used to protect that passphrase.</p>"\
+            "<p>Additional <b>authentication</b> methods can be used for unlocking the device:</p>" \
+            "<ul>" \
+            "<li><i>TPM2: </i>A crypto-device that is already present in your system.</li>" \
```

Comment:
Password authentication is not included here? BTW, please see https://github.com/yast/yast-storage-ng/pull/1409/files#r2079249695.

Reply:
Fixed and cleaned up.